driver/controller.go

```diff
@@ -963,3 +973,46 @@ func validateCapabilities(caps []*csi.VolumeCapability) bool {
 
 	return supported
 }
+
+func (d *Driver) tagVolume(ctx context.Context, vol *godo.Volume) error {
+	for _, tag := range vol.Tags {
+		if tag == d.doVolTag {
+			return nil
+		}
+	}
+
+	tagReq := &godo.TagResourcesRequest{
+		Resources: []godo.Resource{
+			godo.Resource{
+				ID:   vol.ID,
+				Type: godo.VolumeResourceType,
+			},
+		},
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, doAPITimeout)
+	resp, err := d.tags.TagResources(ctx, d.doVolTag, tagReq)
```

Comment:
I would return immediately here instead of returning below. It makes the code easier to read.